application/src/main/java/org/thingsboard/server/service/install/SqlTsDatabaseSchemaService.java

```diff
@@ -36,5 +41,21 @@ public SqlTsDatabaseSchemaService() {
     public void createDatabaseSchema() throws Exception {
         super.createDatabaseSchema();
         executeQuery("CREATE TABLE IF NOT EXISTS ts_kv_indefinite PARTITION OF ts_kv DEFAULT;");
+
+        Long schemaVersion = jdbcTemplate.queryForList("SELECT schema_version FROM tb_schema_settings", Long.class).stream().findFirst().orElse(null);
+
+        if (schemaVersion == null) {
+            jdbcTemplate.execute("INSERT INTO tb_schema_settings (schema_version) VALUES (" + getSchemaVersion() + ")");
```

Comment:
add a try catch for all new code and write meaningful warning with `buildProperties.getVersion()` if something goes wrong.